app/views/reports/_distribution.html.erb

```diff
@@ -1,10 +1,10 @@
 <br>
-<div class="pull-left distribution">
+<div class="pull-left distribution" id="distribution-<%= distribution.id %>">
```

Comment:
Curious why this change was made?

Reply:
Its removed, it was for the system tests but since they are deleted it is deleted too